play-java-starter-example/app/views/main.scala.html

```diff
@@ -4,16 +4,16 @@
  * two arguments, a `String` for the title of the page and an `Html`
  * object to insert into the body of the page.
  *@
-@(title: String)(content: Html)
+ @(title: String, assetsFinder: AssetsFinder)(content: Html)
```

Comment:
Make `AssetsFinder` implicit here? I thought that is why you made an `implicit` in `index.scala.html` above.

Reply:
Good catch. I focused in bringing the Java sample in sync with the Scala one. The issue you highlighted is also present in the Scala sample. I'll have a look at necessary changes (since `main` is also used elsewhere) and propagate/sync to the PR of the docs.